README.md

```diff
@@ -34,7 +36,10 @@ Currently, seven `basic` types of data are supported:
     3. an url
 - `ImageLabelManifest`: encapsulates one single image-level annotation
 - `CategoryManifest`: encapsulates the information about a category, such as its name and super category, if applicable
-- `VisionDataset` is an iterable dataset class that consumes the information from `DatasetManifest`.
+- `MultiImageLabelManifest`: encapsulates one multi-image annotation, it contains the referred image indices and labels.
```

Comment:
It is probably easier/clearer to have a table now, with left side being single-image annotation related classes, and right side being multi-image annotation related classes.

Reply:
Since some classes can be used either in single or mult-image, I created a table with the first column marking this information